src/opm/input/eclipse/Schedule/KeywordHandlers.cpp

```diff
@@ -2020,6 +2021,23 @@ Well{0} entered with disallowed 'FIELD' parent group:
         sched_state.pavg.update(std::move(wpave));
     }
 
+    void Schedule::handleWVFPEXP(HandlerContext& handlerContext) {
+        for (const auto& record : handlerContext.keyword) {
```

Comment:
This does not appear to handle the case of the `WVFPEXP` keyword occurring in an `ACTIONX` block.  I'm okay with postponing that situation until the need arises.

Reply:
Do we have a "not supported in ACTIONX block" list

Reply:
> Do we have a "not supported in ACTIONX block" list

I dont' think so, but that would be a good thing to have.